Cannot find namespace 'stylis' / Vite

Hi,
I'm trying to configure Styled Components V6 with Vite and Storybook and during the build, I get the below error:
```
node_modules/styled-components/dist/models/StyleSheetManager.d.ts:58:21 - error TS2503: Cannot find namespace 'stylis'.
58     stylisPlugins?: stylis.Middleware[];
```

Any solution? Thanks!

--- a/packages/styled-components/src/models/StyleSheetManager.tsx
+++ b/packages/styled-components/src/models/StyleSheetManager.tsx
@@ -1,5 +1,6 @@
 import React, { useContext, useEffect, useMemo, useState } from 'react';
 import shallowequal from 'shallowequal';
+import type stylis from 'stylis';
 import StyleSheet from '../sheet';
 import { ShouldForwardProp, Stringifier } from '../types';
 import createStylisInstance from '../utils/stylis';

